README.md

```diff
@@ -464,7 +464,7 @@ your native language(e.g., display `10 Sep` instead of `2020-09-10`).
 A plugin may add or remove any fields in resulting JSON, that allows plugins to communicate. The only
 requirement is that the result should include all fields above.
 
-**Notes**
+### Notes
```

Comment:
It is notes about "Custom formatting" but `###` moves it to the same level. I think it should be nested with `####`

Reply:
Sure. Will update the pull request.